### ISSUE TITLE
Default route possible?

### ISSUE THREAD

**Issue by grempe** [tsndr/cloudflare-worker-router#17]
Is it possible to have a default route that, for example, would return a 404 for any request that does not match an existing route?

e.g. in Express

```js
app.get(/.*/, (req, res) => {
  res.status(404).send();
  //res.json({ status: ":(" });
});
```

https://stackoverflow.com/questions/37988122/404-fallback-route-in-express

**Comment by tsndr** [comment]
That's a great idea, currently it's hardcoded to 404 right [here](https://github.com/tsndr/cloudflare-worker-router/blob/main/index.js#L388). This goes onto my todo list, thank you :)

**Comment by grempe** [comment]
Actually, it's not a 404 that gets returned by default. Its a `204` No Content which is confusing.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

**Comment by tsndr** [comment]
I'm sorry, I linked to the wrong line, but it's still `404`, this is the correct line right [here](https://github.com/tsndr/cloudflare-worker-router/blob/main/index.js#L359).

**Comment by grempe** [comment]
Hi. You closed this issue but I don't see how the recent release addresses this and there seems to be no update to the README.

Did I miss something? Thx

**Comment by tsndr** [comment]
Oh, I was thinking that you just wanted `404` to be returned as default, which is already the case and since you didn't ansewer I closed it.

**Comment by tsndr** [comment]
Here you go :) #19

**Comment by tsndr** [comment]
Released with [v1.3.7](https://github.com/tsndr/cloudflare-worker-router/releases/tag/v1.3.7)
